pkg/proto/aksnodeconfig/v1/helper.go

```diff
@@ -99,6 +103,30 @@ func (nBCB *NBContractBuilder) ValidateNBContract() error {
 	return nil
 }
 
+func (nBCB *NBContractBuilder) GetNodeBootstrapping() (*datamodel.NodeBootstrapping, error) {
```

Comment:
nit: `nCBC` -> `b`

Reply:
will leave it to Devin, he's going to make a follow up PR to sync the naming to match the new convention of `AKSNodeConfig`, e.g. `AKSNodeConfigBuilder`